loader/checkpoint_test.go

```diff
@@ -54,6 +68,14 @@ func (t *testCheckPointSuite) SetUpSuite(c *C) {
 		MetaSchema: "test",
 	}
 	t.cfg.To.Adjust()
+
+	schemaCreateSQL = fmt.Sprintf("CREATE SCHEMA IF NOT EXISTS `%s`", t.cfg.MetaSchema)
+	tableCreateSQL = fmt.Sprintf("CREATE TABLE IF NOT EXISTS `%s`.`%s` .*", t.cfg.MetaSchema, cputil.LoaderCheckpoint(t.cfg.Name))
+	clearCheckPointSQL = fmt.Sprintf("DELETE FROM `%s`.`%s` WHERE `id` = .*", t.cfg.MetaSchema, cputil.LoaderCheckpoint(t.cfg.Name))
+	loadCheckPointSQL = fmt.Sprintf("SELECT `filename`,`cp_schema`,`cp_table`,`offset`,`end_pos` from `%s`.`%s` where `id`.*", t.cfg.MetaSchema, cputil.LoaderCheckpoint(t.cfg.Name))
+	countCheckPointSQL = fmt.Sprintf("SELECT COUNT.* FROM `%s`.`%s` WHERE `id` = ?", t.cfg.MetaSchema, cputil.LoaderCheckpoint(t.cfg.Name))
+	flushCheckPointSQL = fmt.Sprintf("INSERT INTO `%s`.`%s`  .* VALUES.*", t.cfg.MetaSchema, cputil.LoaderCheckpoint(t.cfg.Name))
```

Comment:
Are two spaces fine here?

Reply:
seems one or two space both ok here, beacuse raw sql is 
[INSERT INTO `test`.`_loader_checkpoint` (`id`, `filename`, `cp_schema`, `cp_table`, `offset`, `end_pos`) VALUES(?,?,?,?,?,?)]